doc/source/whatsnew/v1.5.0.rst

```diff
@@ -500,6 +500,7 @@ Conversion
 - Bug in :func:`array` with ``FloatingDtype`` and values containing float-castable strings incorrectly raising (:issue:`45424`)
 - Bug when comparing string and datetime64ns objects causing ``OverflowError`` exception. (:issue:`45506`)
 - Bug in metaclass of generic abstract dtypes causing :meth:`DataFrame.apply` and :meth:`Series.apply` to raise for the built-in function ``type`` (:issue:`46684`)
+- Bug in :meth:`DataFrame.to_dict` for ``orient=="list"`` or ``orient=="index"`` was not returning native types. (:issue:`46751`)
```

Comment:
nit: specify as the user would when calling the method; e.g. `orient="list"`. Also no period before the issue #.

Reply:
@rhshadrach done